dispatcher/backend/src/routes/schedules/schedule.py

```diff
@@ -76,6 +76,10 @@ def post(self, token: AccessToken.Payload):
         except ValidationError as e:
             raise InvalidRequestJSON(e.messages)
 
+        # make sure name doesn't contain whitespaces around it
```

Comment:
this should be part of schema validation

Reply:
Done